client/src/components/CategoryShow.js

```diff
@@ -1,14 +1,34 @@
 import React, { useState, useEffect } from "react";
 import PastaTile from "./PastaTile.js";
+import NewPastaForm from "./NewPastaForm.js";
+import ErrorList from './ErrorList'
+import translateServerErrors from '../services/translateServerErrors'
+import getCurrentUser from "../services/getCurrentUser.js";
 
 const CategoryShow = (props) => {
   const [category, setCategory] = useState({
     name: "",
-    pastas: [],
+    pastas: []
   });
+  const [errors, setErrors] = useState([])
+  
+  const [currentUser, setCurrentUser] = useState(undefined);
+  
+  const fetchCurrentUser = async () => {
```

Comment:
The current user is already fetch in the App component, so we don't need another fetch request here. Instead, we can pass the currentUser to this component if it's really needed